tests/metrics/functional/test_classification.py

```diff
@@ -45,25 +45,30 @@
 ])
 def test_against_sklearn(sklearn_metric, torch_metric):
     """Compare PL metrics to sklearn version."""
-    pred = torch.randint(10, (500,))
-    target = torch.randint(10, (500,))
+    device = 'cuda' if torch.cuda.is_available() else 'cpu'
```

Comment:
the lightning module has a self.device property...

Also, we don't know if user will always be on cuda or not... so this should not depend on that.
Whether a metric needs cuda, tpu or cpu support should come from lightning or a module but not the user.

Reply:
this is just for test